### ISSUE TITLE
Compiler version not fixed in role contracts

### ISSUE THREAD

**Issue by bearcanrun** [Superalgos/ALGOToken#18]
In AlgoCoreTeamRole.sol(L:1), AlgoSystemRole.sol(L:1), AlgoSupervisorRole.sol(L:1), exact compiler version is not used.

Future compiler versions may handle certain language constructions in a way the developer did not foresee. Thus it is best practice in solidity to indicate exact compiler version.

**Code:**

```
# AlgoCoreTeamRole.sol(L:1), AlgoSystemRole.sol(L:1), AlgoSupervisorRole.sol(L:1)

pragma solidity ^0.4.24;

// Recommended
pragma solidity 0.4.24;

```

**Comment by pmmax** [comment]
Fixed in #21

**Comment by bearcanrun** [comment]
Documentation updated.
